src/validator.cpp

```diff
@@ -260,8 +260,13 @@ struct Validator::ValidatorImpl
                             std::vector<std::vector<std::string>> &errorList);
 
     /**
-    * @brief checks model mapping to ensure all variables mapped to other variables have the same units.
+    * @brief Checks @p model mappings to ensure all variables mapped to other variables have the same units.
+    * Any errors are logged in the @c Validator.
```

Comment:
Brief statements can only have one sentence just delete the second sentence you mention it in the full description.